pkg/nftlabs/sdk.go

```diff
@@ -208,7 +218,7 @@ func (sdk *Sdk) TransferNativeToken(to string, amount *big.Int) error {
 		return err
 	}
 
-	gasLimit := uint64(21000)                // in units
```

Comment:
@jakeloo I don't remember why we set this strict gas limit, do we want to keep this in?